src/ngAria/aria.js

```diff
@@ -235,30 +251,24 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
           }
         },
         post: function(scope, elem, attr, ngModel) {
-          var needsTabIndex = shouldAttachAttr('tabindex', 'tabindex', elem)
-                                && !isNodeOneOf(elem, nodeBlackList);
+          var needsTabIndex = shouldAttachAttr('tabindex', 'tabindex', elem, false);
+          var ngTrueValue = $parse(attr.ngTrueValue)(scope);
 
           function ngAriaWatchModelValue() {
             return ngModel.$modelValue;
           }
 
           function getRadioReaction() {
-            if (needsTabIndex) {
-              needsTabIndex = false;
-              return function ngAriaRadioReaction(newVal) {
-                var boolVal = (attr.value == ngModel.$viewValue);
-                elem.attr('aria-checked', boolVal);
-                elem.attr('tabindex', 0 - !boolVal);
-              };
-            } else {
-              return function ngAriaRadioReaction(newVal) {
-                elem.attr('aria-checked', (attr.value == ngModel.$viewValue));
-              };
-            }
+            return function ngAriaRadioReaction(newVal) {
+              var boolVal = (attr.value == ngModel.$viewValue);
+              elem.attr('aria-checked', boolVal);
+            };
           }
-
-          function ngAriaCheckboxReaction() {
-            elem.attr('aria-checked', !ngModel.$isEmpty(ngModel.$viewValue));
+          function getCheckboxReaction() {
+            return function ngAriaCheckboxReaction(newVal) {
+              var modelValue = attr.ngTrueValue ? ngModel.$viewValue == ngTrueValue : newVal;
```

Comment:
Wow, `mdCheckbox` does some [really hacky stuff][1] and applies the `input[type="checkbox"]` pre-linking function. That's how they get `ngTrue/FalseValue` to work for them.
Pretty interesting...

Anyway, the current implementation works with `ngTrue/FalseValue`, so we're good :)

[1]: https://github.com/angular/material/blob/ad9ba5277651423beceb7302197e069047a5105d/src/components/checkbox/checkbox.js#L102-L108